proposals/wac-ucr/index.bs

```diff
@@ -962,6 +1357,12 @@ over 18. Her client's credential manager should see this and select the
 credential revealing the least amount of information needed to access the
 resource.
 
+<div class="assertion">
+
+* [[#req-vc]]
+ 
+</div>
+
 ### Limit information disclosure through URI ### {#uc-limituri}
 
 A service with very high security and confidentiality requirements must
```

Comment:
The requirement is missing here.
The Requirement is:
 "The system shall not require inspection of URL to decide on the type of resources, be they containers, ACRs or Resources"

Reply:
Just a small clarification, does it affect also the distinction between ldp resources of type container and other types?

> Servers MUST create a container with URI path ending /{id}/ in container / for requests including the HTTP Link header with rel="type" targeting a valid LDP container type.
>
> — Solid protocol spec: https://solidproject.org/TR/protocol#writing-resources

Reply:
I agree with you that there should be no semantics in the URIs but it seems like it's not strictly the case in the spec currently.

Reply:
As far as possible the specs should use nice humanly readable URLs where resources ending in `/` refer to containers and others to `ldp:Resource`. That makes the spec much easier to read. And indeed any public facing non paranoid service  (eg blogs, universities, etc...) should use those too, as they are easier to understand and map to intuitions on the file system. 
The requirement is just that the protocol should not rely on these, as that allows us then to put an Solid server behind TOR as described in the use case and have clients work on those TOR-Solid servers just as well.

Reply:
so yes, I think that the spec should not use MUST, but rather SHOULD.

Reply:
LDP does not have that requirement on URLs. I suggested that when those requirements are met we call the container an `iContainer` for intuitive container. See [Enable full use of relative URLs in Solid](https://github.com/solid/specification/issues/194)

Reply:
This is specifically related to the semantics of the slash to imply containment, which is a solid thing, not LDP. I'm trying to construct a requirement that skirts the line on this, rather than revisit what has already been heavily debated elsewhere. At the moment it may be better to avoid adding a requirement for this, and revisit instead during the spec phase.

Reply:
I don't think we need to revise anything as I pointed out above, this should be compatible with how solid works. We just need to be a bit more general that `solid:Containers` or `iContainers` to address this use case. But that is not a big deal: we just need to have resources point to their container using a `Link` relation. (Or something of that sort).
I left a summary here: https://github.com/solid/authorization-panel/pull/152#pullrequestreview-585961054

Reply:
Resolved in https://github.com/solid/authorization-panel/pull/152/commits/1150c90f4aade028dce93821a2a47e3bb6a43e57